app/models/validations/soft_validations.rb

```diff
@@ -97,6 +97,42 @@ def net_income_higher_or_lower_text
     net_income_in_soft_max_range? ? "higher" : "lower"
   end
 
+  def scharge_over_soft_max?
+    return unless scharge && period && needstype
+    return if weekly_value(scharge).blank?
+
+    max = if needstype == 1
```

Comment:
are there any situations where nil values here could exist for needstype or provider type? I don't think so but if that does happen we could get some incorrect maxes

Reply:
If needstype isn't answered we wouldn't route to the soft validation anyway, but good point about provider type. Let me check that